testkit/src/main/java/akka/kafka/testkit/internal/AlpakkaKafkaContainer.java

```diff
@@ -32,7 +32,7 @@
   private static final String STARTER_SCRIPT = "/testcontainers_start.sh";
 
   // Align this with testkit/src/main/resources/reference.conf
-  public static final String DEFAULT_CONFLUENT_PLATFORM_VERSION = "5.4.0";
+  public static final String DEFAULT_CONFLUENT_PLATFORM_VERSION = "6.0.0";
```

Comment:
Does our current Testcontainers version work with 6.0.0?
(The compile error makes me believe you haven't tried.)

Reply:
I was premature on the ready to review button.

Reply:
Fixed